src/Compilers/CSharp/Portable/Operations/CSharpOperationFactory.cs

```diff
@@ -35,40 +32,16 @@ public IOperation Create(BoundNode boundNode)
                 return null;
             }
 
-            // this should be removed once this issue is fixed
-            // https://github.com/dotnet/roslyn/issues/21187
-            if (IsIgnoredNode(boundNode))
+            // implicit receiver can be shared between multiple bound nodes.
```

Comment:
> implicit receiver can be shared between multiple bound nodes. [](start = 14, length = 62)

I cannot figure out a scenario when this can happen. Could you provide more details on the scenario.

Reply:
you can live repro by running TestClone or TestParentOperation, but one of example is

var o4 = new MyObject(@dynamic)
            {
                A = 0,
                B = 0,
                C = 0
            };

where A, B, C is none operation and its child point to same instanceReferenceExpression to error type MyObject